### ISSUE TITLE
Funnel chart: enable sorting differently

### ISSUE THREAD

**Issue by magnew** [lightdash/lightdash#10498]
### Description

Funnel chart currently always sorts descending. Add an options for:
- No sorting; keep the order of the columns
- Ascending sort

**Comment by hamzahc1** [comment]
Could you also drag the steps in whatever order you want? Particularly useful when using rows rather than columns

**Comment by magnew** [comment]
> Could you also drag the steps in whatever order you want? Particularly useful when using rows rather than columns

Yup. Updated description. We had a customer request for this too.

**Comment by jpetey75** [comment]
customer request here from july 2024 https://lightdash.slack.com/archives/C05F0N3UPSP/p1720519918140639
